index.html

```diff
@@ -11,13 +11,13 @@
       <!--Import Google Icon Font-->
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <!--Import materialize.css-->
-      <link type="text/css" rel="stylesheet" href="./static/materialize/css/materialize.min.css"  media="screen,projection"/>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">

```

Comment:
Hi @xit4, nice job there! But can you please explain why did you remove link type and media? :smile:

Reply:
I used the "suggested" <link> from the materialize [getting started page](https://materializecss.com/getting-started.html)

Anyway, this is how it would look like with those attributes.
```suggestion
      <link rel="stylesheet" type="text/css" media="screen,projection" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```

Reply:
Ok, no problem!